bundles/org.openhab.binding.renault/README.md

```diff
@@ -69,24 +71,24 @@ renaultcar.sitemap:
 sitemap renaultcar label="Renault Car" {
```

Comment:
I removed the binary image of the site map as requested but kept the text one.  This gives uses an easy UI but I think this could be improved with the new openhab UI 3 in future.  I kept the icons because the category definitions in the channel XML caused me runtime errors on start.

Reply:
Which error did you get? Are you sure you put the category at the right position (order in XML has to be correct according to XSD)?

Reply:
Ah OK.  That might have been the issue.  I will try again.

Reply:
@jlaur I have added the categories after description and this is working.  If the Items are re-created they adopt the category Time and icon in the site map.  I have remove the redundant icon definition from the site map as suggested.